test/library/packages/DistributedBag/add.good

```diff
@@ -0,0 +1,4 @@
+add.chpl:17: warning: maximum capacity reached: some elements may have been missed.
+add.chpl:17: warning: maximum capacity reached: some elements may have been missed.
+add.chpl:17: warning: maximum capacity reached: some elements may have been missed.
```

Comment:
what is printing this warning? Generally, it's best to avoid a warning printed from a library, and rely on other mechanisms instead. Is there anything we could do instead of this warning? I thought that `add` would return a smaller number if something is not added? Could we argue that the warning isn't necessary and it's rather the calling code (the test in this case) that should be checking the number returned by `bag.add`?

Reply:
I strengthened the test by checking that `add` returns `true` when it is expected, and by checking that we get the good number of failures.